packages/create-cloudflare/templates-experimental/react/ts/wrangler.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "<TBD>",
+	"main": "api/index.ts",
+  "compatibility_date": "<TBD>",
+	"assets": { "not_found_handling": "single-page-application", "binding": "ASSETS" },
+  "observability": {
+    "enabled": true
+  }
+}
```

Comment:
this file contains indentation done both via spaces and tabs

Reply:
ebbe56150836ae4953f9b77ccdcea63a1d3402fd